app/harvesters/loc_harvester.rb

```diff
@@ -0,0 +1,196 @@
+    super
+  end
+
+  ##
+  # @return [Hash] A hash documenting the allowable options to pass to
+  #   initializers.
+  #
+  # @see Krikri::Harvester::expected_opts
+  def self.expected_opts
+    { 
+      key: :api,
+      opts: {
+        uris:          { type: :string, required: true,  multiple_ok: true},
+        item_base_uri: { type: :string, required: false, multiple_ok: false}
+      }
+    }
+  end
+
+  ##
+  # @param identifier [#to_s] the identifier of the record to get
+  # @return [#to_s] the record
+  def get_record(identifier)
+    build_record(request(build_record_uri(identifier)))
+  end
+
+  ##
+  # @see Harvester#record_ids
+  def record_ids
+    enumerate_uris.map { |u| u.split('/').last }.to_enum
+  end
+
+  private
+
+  ##
+  # @private
+  def build_record_uri(identifier)
+    RDF::URI(opts[:item_base_uri]) / identifier.to_s / JSON_PARAM
+  end
+
+  ##
+  # Overrides ApiHarvester.get_identifier because the idenitifer field
+  # for Library of Congress
+  #
+  # @param doc [Hash] a JSON serialziation with an identifier
+  #
+  # @return [String] the provider's identifier for the document
+  def get_identifier(doc)
+    # doc is the response to a request for /item/<id>
+    doc['item']['library_of_congress_control_number']
+  end
+
+  ##
+  # @param response [Hash] a response from the LoC API
+  #
+  # @return [Integer] value of the number of records in the response
+  def get_count(response)
+    response['pagination']['of']
+  end
+
+  ##
+  # @param response [Hash] a response from the LoC API
+  #
+  # @return [Array] the documents in the response
+  def get_docs(response)
+    response['results']
+  end
+
+  ##
+  # Get the URL for the next request.
+  #
+  # @param opts [Hash] the response to the current request. The next page is
+  # provided in the ['next'] element
+  #
+  # @param response [Hash]
+  #
+  # @return [Hash] the next request's options hash
+  def next_options(response)
+    response['pagination']['next']
+  end
+
+  ##
+  # @return [Enumerator] an enumerator over the records
+  def enumerate_records
+    enumerate_uris.lazy.map do |item_uri|
+      @http.add_request(uri: URI.parse(item_uri + JSON_PARAM))
+        .with_response do |item_response|
+        Krikri::Logger.log(:error, "#{response.status}: #{uri}") unless
+          item_response.status == 200
+        
+        parse_json(item_response.body)
+      end
+    end
+  end
+
+  ##
+  # @return [Enumerator] an enumerator over the record uris
+  def enumerate_uris
+    Enumerator.new do |yielder|
+      # For each of the sets
+      opts[:uris].each do |uri|
+        loop do
+          response = request(uri)
+          batch    = get_docs(response)
+
+          next  if batch.nil?
+          break if batch.empty?
+          
+          batch.each do |item|
+            item_uri = Array.new([item['aka'], item['id'], item['url']])
+                       .flatten.uniq.grep(/www.loc.gov\/item/).first
```

Comment:
It's definitely possible, but I wasn't sure it was worth the time. It might be that we should just remove `item_base_uri` as an option, unless we think they are going to vary on LoC's side.